pkgs/development/libraries/glibc/common.nix

```diff
@@ -138,7 +139,7 @@ stdenv.mkDerivation ({
 
   depsBuildBuild = [ buildPackages.stdenv.cc ];
   nativeBuildInputs = [ bison ];
-  buildInputs = [ linuxHeaders ] ++ lib.optionals withGd [ gd libpng ];
+  buildInputs = [ linuxHeaders python3 ] ++ lib.optionals withGd [ gd libpng ];
```

Comment:
Is the interpreter used during the build? If so, you want to move it to `nativeBuildInputs` instead.